### PR TITLE
Trust user added CAs on Android7+

### PR DESCRIPTION
**Pull request by AndyScherzinger**
See https://android-developers.googleblog.com/2016/07/changes-to-trusted-certificate.html for further details.

With Android 7 Google changed the handling of CAs:
> Apps that target API Level 24 and above no longer trust user or admin-added CAs for secure connections, by default. 

This change would basically lead to trusting used added CAs. Please have a look @tobiasKaminsky @mario and especially @LukasReschke if this is something we want. If not, feel free to close the PR and delete the branch.

### CODE REVIEW

**Comment by mario** [comment]
Looks decent (except for conflicts) - but why do you kill animations here? o.o

File: src/main/res/xml/network_security_config.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<network-security-config>
+    <base-config cleartextTrafficPermitted="true">
+        <trust-anchors>
+            <!-- Trust preinstalled CAs -->
+            <certificates src="system"/>
+            <!-- Additionally trust user added CAs -->
+            <certificates src="user"/>
+        </trust-anchors>
+    </base-config>
+</network-security-config>
```
Comment (LukasReschke) [inline src/main/res/xml/network_security_config.xml]: Looks good but no idea about all the other changes to the anim files above :)

**Comment by AndyScherzinger** [comment]
@mario I removed the animations due to lint bragging about "unused resources" 😀

**Comment by mario** [comment]
In this PR that has nothing to do with it? :D Peeeerfect.

Just resolve conflicts.

**Comment by AndyScherzinger** [comment]
> In this PR that has nothing to do with it? :D Peeeerfect.

Yes and no. Lint score got worse and at the moment it is not possible to see what changed the score so I simply fixed another issue to get a green score again ;)

**Comment by mario** [comment]
👍 

@LukasReschke approve.

**Pull request merged**